### PR TITLE
created .env and .env.example file and schedule BMC link 

### PR DESCRIPTION
**Pull request by rohitkori**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -85,17 +87,22 @@ const exampleEmbed = new Discord.MessageEmbed()
...
 rule.hour = 10;
 rule.minute = 00;
 var startDay = 0;
 var endDay = 6;
 
-//creating a empty cron expression
-var cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+const ruleForBMC = new schedule.RecurrenceRule();
+ruleForBMC.dayOfWeek = [0, new schedule.Range(0, 6, 2)];
+ruleForBMC.hour = 12;
```
Comment (RahulGopathi) [inline app.js]: since these are same for every server, how about adding these to `.env`?

File: app.js [hunk 2]
```diff
@@ -123,6 +130,28 @@ function fetchRandomArticle(category) {
...
+  console.log(ruleForBMC);
+  const job = schedule.scheduleJob(ruleForBMC, function () {
+    client.guilds.cache.each((guild) => {
+      try {
+        const channel =
+          guild.channels.cache.find(
+            (channel) => channel.name === "readsomethinggreat"
+          ) || guild.channel.cache.first();
+        if (channel) {
+          channel.send("**Buy me a Coffee link**- " + "https://www.buymeacoffee.com/rahulgopathi");
```
Comment (RahulGopathi) [inline app.js]: we can also make this BMC link a variable

File: app.js [hunk 3]
```diff
@@ -123,6 +130,28 @@ function fetchRandomArticle(category) {
   return articleLink;
 }
 
+function BMCLinkScheduler() {
+  console.log(ruleForBMC);
```
Comment (RahulGopathi) [inline app.js]: you can remove this print statement

**Review by RahulGopathi — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (rohitkori) [reply app.js]: > we can also make this BMC link a variable

yes , i will do it

File: app.js (hunk 3 above)
Comment (rohitkori) [reply app.js]: okk i will

File: app.js (hunk 1 above)
Comment (rohitkori) [reply app.js]: Ok, I will try doing it

**Review by RahulGopathi — APPROVED** [review]
LGTM

**Pull request merged**